src/ucan/app.ts

```diff
@@ -0,0 +1,5 @@
+export const PREFIX = "app"
+
+export type Capability
+  = "DESTROY"
+  | "REVISE"
```

Comment:
`DESTROY` is now `SUPER_USER` for consistency (as of like 20 minutes ago 😅)

Reply:
SUPER_USER also gives you the ability to do things like transfer ownership, which is essentially destroy+create (i.e. "move" to another user)